docker/iota-data-ingestion/build.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 # Copyright (c) 2024 IOTA Stiftung
 # SPDX-License-Identifier: Apache-2.0
-./../utils/build-script.sh --container-name "iotaledger/iota-data-ingestion"
+./../utils/build-script.sh --image-tag "iotaledger/iota-data-ingestion"
```

Comment:
This makes the script depend on being in a particular dir when running it.
Can we make it work regardless of the current directory ?
For instance with:
```suggestion
REPO_ROOT="$(git rev-parse --show-toplevel)"

DOCKERFILE_DIR="$REPO_ROOT/docker/iota-data-ingestion" \
"$REPO_ROOT/docker/utils/build-script.sh" --image-tag "iotaledger/iota-data-ingestion"
```


and make `docker/utils/build-script.sh` accept the override:
```
CURRENT_WORKING_DIR="$(pwd)"
DOCKERFILE_DIR="${DOCKERFILE_DIR:-$CURRENT_WORKING_DIR}"
```

Reply:
that's a good point about needing git. So okay for leaving this for a later issue/PR.